bridge/client/app/_components/ktb-event-item/ktb-event-item.component.html

```diff
@@ -16,16 +16,16 @@ <h4 class="m-0 mt-1 mb-1" [class.error]="_event.isFaulty()" [textContent]="_even
         <p class="m-0 nobreak"><span class="bold">Problem: </span><span [textContent]="_event.getProblemDetails()"></span></p>
       </div>
     </div>
-    <div fxLayout="column" fxLayoutAlign="start end">
+    <div fxLayout="column" fxLayoutAlign="start end" uitestid="keptn-event-item-contextButtons">
       <div fxLayout="row" fxLayoutAlign="start start">
         <p *ngIf="showTime" class="m-0 mt-1 mb-1" [textContent]="_event.time | amDateFormat:dateUtil.getDateTimeFormat()"></p>
-        <button dt-icon-button variant="nested" [routerLink]="['/evaluation', _event.shkeptncontext, _event.getStage()]" *ngIf="_event.isEvaluation() && _event.isFinished()"><dt-icon name="chart-bar"></dt-icon></button>
-        <button dt-icon-button variant="nested" (click)="showEventPayloadDialog()"><dt-icon name="coding"></dt-icon></button>
+        <button dt-icon-button variant="nested" uitestid="keptn-event-item-contextButton-evaluation" [routerLink]="['/evaluation', _event.shkeptncontext, _event.getStage()]" *ngIf="_event.isEvaluation() && _event.isFinished()"><dt-icon name="chart-bar"></dt-icon></button>
+        <button dt-icon-button variant="nested" uitestid="keptn-event-item-contextButton-payload" (click)="showEventPayloadDialog()"><dt-icon name="coding"></dt-icon></button>
       </div>
     </div>
   </div>
   <div *ngIf="_event.data.problem?.ProblemDetails">
-    <div fxLayout="row" fxLayoutAlign="start start" *ngIf="_event.data.problem.ProblemDetails.tagsOfAffectedEntities">
+    <div fxLayout="row" fxLayoutAlign="start start" *ngIf="_event.data.problem.ProblemDetails.tagsOfAffectedEntities" uitestid="keptn-event-item-tags">
```

Comment:
Change this to keptn-event-item-problemTags to give it more context

Reply:
Done